src/app/treasure-hunt/treasure-hunt-calculator-services/boiler-blowdown-rate-treasure-hunt.service.ts

```diff
@@ -0,0 +1,200 @@
+            percentSavings: energyData.percentSavings,
+            utilityType: energyData.utilityTypes,
+            boilerBlowdownRate: opportunity,
+            name: opportunitySummary.opportunityName,
+            opportunitySheet: opportunity.opportunitySheet,
+            iconString: 'assets/images/calculator-icons/steam-icons/blowdown-rate.png',
+            teamName: opportunity.opportunitySheet ? opportunity.opportunitySheet.owner : undefined,
+            iconCalcType: 'steam',
+            needBackground: true
+        }
+        return cardData;
+    }
+
+    getEnergyData(boilerBlowdownRate: BoilerBlowdownRateTreasureHunt, settings: Settings, currentEnergyUsage: EnergyUsage, opportunitySummary: OpportunitySummary): {
+        annualEnergySavings: Array<{
+            savings: number,
+            label: string,
+            energyUnit: string
+        }>,
+        percentSavings: Array<{ percent: number, label: string, baselineCost: number, modificationCost: number }>,
+        utilityTypes: Array<string>
+    } {
+        let annualEnergySavings: Array<{ savings: number, label: string, energyUnit: string }> = new Array();
+        let percentSavings: Array<{ percent: number, label: string, baselineCost: number, modificationCost: number }> = new Array();
+        let utilityTypes: Array<string> = new Array();
+        let baselineResults: BoilerBlowdownRateResults = this.boilerBlowdownRateService.calculateResults(boilerBlowdownRate.baseline, settings, true, true);
+        let modificationResults: BoilerBlowdownRateResults = this.boilerBlowdownRateService.calculateResults(boilerBlowdownRate.modification, settings, true, true);
+        
+        let currentCosts: number = 0;
+        if (boilerBlowdownRate.baseline.boilerUtilityType == 'Natural Gas') {
+            currentCosts = currentEnergyUsage.naturalGasCosts;
+        } else if (boilerBlowdownRate.baseline.boilerUtilityType == 'Other Fuel') {
+            currentCosts = currentEnergyUsage.otherFuelCosts;
+        }
+
+        let fuelUnit: string = 'MMBTu/yr';
+        if (settings.unitsOfMeasure == 'Metric') {
+            fuelUnit = 'MJ/yr';
+        }
+        annualEnergySavings.push({
+            savings: baselineResults.boilerFuelUse - modificationResults.boilerFuelUse,
+            label: boilerBlowdownRate.baseline.boilerUtilityType,
+            energyUnit: fuelUnit
+        });
+        percentSavings.push(
+            {
+                percent: ((baselineResults.boilerFuelCost - modificationResults.boilerFuelCost) / currentCosts) * 100,
+                label: boilerBlowdownRate.baseline.boilerUtilityType,
+                baselineCost: baselineResults.boilerFuelCost,
+                modificationCost: modificationResults.boilerFuelCost
+            }
+        )
+        utilityTypes.push(boilerBlowdownRate.baseline.boilerUtilityType);
+
+        let waterUnit: string = 'L/yr';
+        if (settings.unitsOfMeasure == 'Metric') {
```

Comment:
I thin L/yr would be the metric value. If so, this should be reversed. More importantly, I don't really see where the 'energyUnit' value is being used in TH. It appears units are set at the display level, not at TH service level. So it's possible this code block is a meme that was once in use, or for a sub set of calculators but no longer is. Let me know what you find